### PR TITLE
Release Notes for 2023-06-25

### PR DESCRIPTION
**Pull request by justindavies**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,37 @@
...
+* Because of Ubuntu 22.04 FIPS certification status, we'll [switch AKS FIPS nodes from 18.04 to 20.04](https://learn.microsoft.com/azure/aks/supported-kubernetes-versions?tabs=azure-cli#aks-components-breaking-changes-by-version) from 1.27 preview onwards.
+* After May 31, 2023, Ubuntu 18.04 will reach end of life. AKS will continue to update the host OS from Canonical into the Kubernetes 1.24 VHD images. Customers will not receive daily security updates from Canonical past the end of May, but will be able to consume those through a node image update only. 
+* Windows2019 will be retired in Kubernetes v1.33 and above (ETA March 2026). Customers should [upgrade to Windows2022](https://learn.microsoft.com/azure/aks/upgrade-windows-2019-2022).
+* Kubernetes 1.24 is being deprecated end of July. From Kubernetes 1.25 the default cgroups implementation on AKS nodes will be cgroupsv2. Older versions of Java, .NET and NodeJS do not support memory querying v2 memory constraints and this will lead to out of memory (OOM) issues for workloads. Please test your applications for cgroupsv2 compliance.
+
+### Release notes
+
+* Preview Features
```
Comment (allyford) [inline CHANGELOG.md:17]: Should remove preview features section if there is nothing to add

Comment (justindavies) [reply CHANGELOG.md:17]: Yeah - thanks - I left it there for the team to add in case there was anything to call out

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,37 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2023-06-25
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
+
+* [Kubernetes 1.24 is the last version of Kubernetes supported by AKS Engine](https://github.com/Azure/aks-engine#project-status). Kubernetes 1.24 goes end-of-life in July, at which point Upstream will stop releasing patches for AKS Engine and archive the project. Please consider using [Azure Kubernetes Service (AKS)](https://azure.microsoft.com/products/kubernetes-service/#overview) for managed Kubernetes or [Cluster API Provider Azure](https://github.com/kubernetes-sigs/cluster-api-provider-azure) for self-managed Kubernetes.
+* Because of Ubuntu 22.04 FIPS certification status, we'll [switch AKS FIPS nodes from 18.04 to 20.04](https://learn.microsoft.com/azure/aks/supported-kubernetes-versions?tabs=azure-cli#aks-components-breaking-changes-by-version) from 1.27 preview onwards.
```
Comment (sabbour) [inline CHANGELOG.md:10]: > Because of Ubuntu 22.04 FIPS certification status

Can we link to this for more context?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,37 @@
...
+* After May 31, 2023, Ubuntu 18.04 will reach end of life. AKS will continue to update the host OS from Canonical into the Kubernetes 1.24 VHD images. Customers will not receive daily security updates from Canonical past the end of May, but will be able to consume those through a node image update only.
```
Comment (sabbour) [inline CHANGELOG.md:11]: > AKS will continue to update the host OS from Canonical into the Kubernetes 1.24 VHD images

Until when? 1.24 EOL?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,37 @@
...
+* Because of Ubuntu 22.04 FIPS certification status, we'll [switch AKS FIPS nodes from 18.04 to 20.04](https://learn.microsoft.com/azure/aks/supported-kubernetes-versions?tabs=azure-cli#aks-components-breaking-changes-by-version) from 1.27 preview onwards.
+* After May 31, 2023, Ubuntu 18.04 will reach end of life. AKS will continue to update the host OS from Canonical into the Kubernetes 1.24 VHD images. Customers will not receive daily security updates from Canonical past the end of May, but will be able to consume those through a node image update only. 
+* Windows2019 will be retired in Kubernetes v1.33 and above (ETA March 2026). Customers should [upgrade to Windows2022](https://learn.microsoft.com/azure/aks/upgrade-windows-2019-2022).
```
Comment (sabbour) [inline CHANGELOG.md:12]: Is writing Windows2019 and Windows2022 without spaces intentional?

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,37 @@
...
+* Kubernetes 1.24 is being deprecated end of July. From Kubernetes 1.25 the default cgroups implementation on AKS nodes will be cgroupsv2. Older versions of Java, .NET and NodeJS do not support memory querying v2 memory constraints and this will lead to out of memory (OOM) issues for workloads. Please test your applications for cgroupsv2 compliance.
```
Comment (sabbour) [inline CHANGELOG.md:13]: > being deprecated end of July

I propose `being deprecated end of July, 2023`.

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,37 @@
...
+
+### Release notes
+
+* Preview Features
+
+* Behavior Changes
+  * Added node anti affinity for Cilium and Azure Linux so that Network Observability extension does not run (not supported environments)
```
Comment (sabbour) [inline CHANGELOG.md:20]: Does this mean that Network Observability extension doesn't support Azure Linux? Doesn't support Cilium? or both? Can you clarify?

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,37 @@
...
+  * Cluster create check that the size of Kubenet based clusters will not exceed 400 nodes (Kubenet on Azure limit)
+
+* Bug Fixes
+  * Enable failure-domain.beta.kubernetes.io labels on K8S 1.26+ nodes by default
```
Comment (sabbour) [inline CHANGELOG.md:24]: Why is this a bug?

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,37 @@
...
+
+* Component Updates
+  * Upgrade Secret store driver to v1.3.4
+  * Upgrade Azure Disk CSI driver version to 1.26.5  on K8S 1.24+
+  * Upgrade AGIC addon to version 1.7.1 for K8S >= 1.27
+  * AKS Ubuntu 18.04 image has been updated to [AKSUbuntu-1804-202306.13.0](vhd-notes/aks-ubuntu/AKSUbuntu-1804/202306.19.0.txt).
```
Comment (shashankbarsin) [inline CHANGELOG.md:30]:
```suggestion
  * AKS Ubuntu 18.04 image has been updated to [AKSUbuntu-1804-202306.19.0](vhd-notes/aks-ubuntu/AKSUbuntu-1804/202306.19.0.txt). 
```

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,5 +1,37 @@
...
+* Bug Fixes
+  * Enable failure-domain.beta.kubernetes.io labels on K8S 1.26+ nodes by default
+
+* Component Updates
+  * Upgrade Secret store driver to v1.3.4
+  * Upgrade Azure Disk CSI driver version to 1.26.5  on K8S 1.24+
+  * Upgrade AGIC addon to version 1.7.1 for K8S >= 1.27
+  * AKS Ubuntu 18.04 image has been updated to [AKSUbuntu-1804-202306.13.0](vhd-notes/aks-ubuntu/AKSUbuntu-1804/202306.19.0.txt). 
+  * AKS Ubuntu 22.04 image has been updated to [AKSUbuntu-2204-202306.13.0](vhd-notes/aks-ubuntu/AKSUbuntu-2204/202306.13.0.txt).
+  * Azure Linux image has been updated to [AzureLinux-202306.13.0](vhd-notes/AzureLinux/202306.19.0.txt).
```
Comment (shashankbarsin) [inline CHANGELOG.md:32]:
```suggestion
  * Azure Linux image has been updated to [AzureLinux-202306.19.0](vhd-notes/AzureLinux/202306.19.0.txt).
```

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,5 +1,37 @@
...
+  * Added node anti affinity for Cilium and Azure Linux so that Network Observability extension does not run (not supported environments)
+  * Cluster create check that the size of Kubenet based clusters will not exceed 400 nodes (Kubenet on Azure limit)
+
+* Bug Fixes
+  * Enable failure-domain.beta.kubernetes.io labels on K8S 1.26+ nodes by default
+
+* Component Updates
+  * Upgrade Secret store driver to v1.3.4
+  * Upgrade Azure Disk CSI driver version to 1.26.5  on K8S 1.24+
+  * Upgrade AGIC addon to version 1.7.1 for K8S >= 1.27
```
Comment (shashankbarsin) [inline CHANGELOG.md:29]: can we add links to each of these releases?

File: CHANGELOG.md (hunk 4 above)
Comment (justindavies) [reply CHANGELOG.md:12]: It's the naming convention used for the OSSKU

File: CHANGELOG.md (hunk 7 above)
Comment (shashankbarsin) [reply CHANGELOG.md:24]: https://github.com/Azure/AKS/issues/3668 - I think we need to share more details on this being a temporary fix until 1.28

File: CHANGELOG.md [hunk 11]
```diff
@@ -1,5 +1,35 @@
...
+
+* Bug Fixes
+  * [Enable failure-domain.beta.kubernetes.io](https://github.com/Azure/AKS/issues/3668) labels on K8S 1.26+ nodes by default to resolve issue with in tree CSI drivers.  Will be removed from K8S 1.28
+
+* Component Updates
+  * Upgrade Secret store driver to [v1.3.4](https://github.com/kubernetes-sigs/secrets-store-csi-driver/releases/tag/v1.3.4)
+  * Upgrade Azure Disk CSI driver version to [1.26.5](https://github.com/kubernetes-sigs/azuredisk-csi-driver/releases/tag/v1.26.5) on K8S 1.24+
+  * Upgrade AGIC addon to version [1.7.1](https://github.com/Azure/application-gateway-kubernetes-ingress/releases/tag/1.7.1) for K8S >= 1.27
+  * AKS Ubuntu 18.04 image has been updated to [AKSUbuntu-1804-202306.19.0](vhd-notes/aks-ubuntu/AKSUbuntu-1804/202306.19.0.txt). 
+  * AKS Ubuntu 22.04 image has been updated to [AKSUbuntu-2204-202306.19.0](vhd-notes/aks-ubuntu/AKSUbuntu-2204/202306.13.0.txt).
```
Comment (shashankbarsin) [inline CHANGELOG.md:29]:
```suggestion
  * AKS Ubuntu 22.04 image has been updated to [AKSUbuntu-2204-202306.19.0](vhd-notes/aks-ubuntu/AKSUbuntu-2204/202306.19.0.txt).
```

**Review by sabbour — APPROVED** [review]

**Review by shashankbarsin — APPROVED** [review]

**Pull request merged**